dataloader/tasks_multilingual.py

```diff
@@ -15,6 +16,11 @@
     get_database
 )
 
+def should_download_file(filename):
+    #if "D0543" in filename:
+    return True
```

Comment:
What is the point of this function if it always returns true??? Is it for debugging purposes only and does it need to stay in?

Reply:
This is just a catch in order to be able to limit the loader to load only one file. Currently this catch is not active this is why the function always returns true. I can delete it if you think its better not to have it in the code!

Reply:
It is better not to have debugging functions in it but if you use it often I will just block out the whole function and its call and then I will merge this branch.